src/gateway/Messages.sol

```diff
@@ -846,6 +840,38 @@ library Messages {
     }
 
     // TODO: should be moved to a util contract
+    function stringToBytes128(string memory source) internal pure returns (bytes memory) {
```

Comment:
Let's adopt the _ pattern for internal functions
```suggestion
    function _stringToBytes128(string memory source) internal pure returns (bytes memory) {
```

Reply:
Want me to update the internal functions in TestSetup.t.sol too?

Reply:
Yes please do!